cmd/merge.go

```diff
@@ -100,42 +100,39 @@ func (m *Merge) mergeProject(
 		writer = changeFile
 	}
 
-	allVersions, err := core.GetAllVersions(cfg, false, project)
+	allVersions, err := cfg.AllVersions(false, project)
 	if err != nil {
 		return fmt.Errorf("finding release notes: %w", err)
 	}
 
-	if cfg.HeaderPath != "" {
-		err = core.AppendFile(writer, filepath.Join(cfg.ChangesDir, cfg.HeaderPath))
+	if cfg.Changelog.Header.FilePath != "" {
+		err = core.AppendFile(writer, filepath.Join(cfg.RootDir, cfg.Changelog.Header.FilePath))
```

Comment:
Increase consistency
```suggestion
	if cfg.ReleaseNotes.Title.FilePath != "" {
		err = core.AppendFile(writer, filepath.Join(cfg.RootDir, cfg.ReleaseNotes.Title.FilePath))
```